examples/customer-segmentation-server/src/mcp-app.css

```diff
@@ -1,42 +1,45 @@
+/* Default fallback values for host style variables */
 :root {
-  --color-bg: #ffffff;
-  --color-text: #1f2937;
-  --color-text-muted: #6b7280;
-  --color-primary: #2563eb;
-  --color-primary-hover: #1d4ed8;
-  --color-card-bg: #f9fafb;
-  --color-border: #e5e7eb;
-  --color-enterprise: #1e40af;
-  --color-midmarket: #0d9488;
-  --color-smb: #059669;
-  --color-startup: #6366f1;
-}
-
-@media (prefers-color-scheme: dark) {
-  :root {
-    --color-bg: #111827;
-    --color-text: #f9fafb;
-    --color-text-muted: #9ca3af;
-    --color-primary: #3b82f6;
-    --color-primary-hover: #60a5fa;
-    --color-card-bg: #1f2937;
-    --color-border: #374151;
-    --color-enterprise: #3b82f6;
-    --color-midmarket: #14b8a6;
-    --color-smb: #10b981;
-    --color-startup: #818cf8;
-  }
+  color-scheme: light dark;
```

Comment:
Changes in this file are me updating this one example to use the host context styles (with its original styles as fallbacks)